alyssa_mckee/Django/restful_users/apps/users/models.py

```diff
@@ -0,0 +1,101 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+import re
+# Create your models here.
+class Deleted_User(models.Model):
```

Comment:
I like to see that you are attempting at implementing soft deletes for the users. This method will work, but instead of moving all the users to a separate table, as this can become cumbersome, then you could implement a deleted_at column on the User model instead. Refer to something like this: https://medium.com/@adriennedomingus/soft-deletion-in-django-e4882581c340